src/admin/settings/settings.html

```diff
@@ -0,0 +1,83 @@
+<legend class="frame-title">{{'admin.platform.settings' | translate}}</legend>
+<form class="settings-form" id="settingsFileForm" enctype="multipart/form-data">
+    <h4>{{'admin.settings.upload' | translate}}</h4>
+    <div class="form-group fileinput fileinput-new" data-provides="fileInput">
```

Comment:
@mkruszynski change data-provides to `"fileinput"` - I think this is causing problems on the UI